src/htmx.js

```diff
@@ -2022,6 +2023,12 @@ return (function () {
                             swapSpec["show"] = showVal;
                             swapSpec["showTarget"] = selectorVal;
                         }
+                        if (modifier.indexOf("focus-scroll:") === 0) {
+                            var focusScrollSpec = modifier.substr("focus-scroll:".length);
+                            var splitSpec = focusScrollSpec.split(":");
```

Comment:
do we need to split here?  It looks like you'd just have the value `true`

Reply:
The modifier value could also be `false` when the global config is `true`, but you want to change set it as `false` for a particular request using the modifier.

Reply:
Right, but the 

```js
var splitSpec = focusScrollSpec.split(":");
```

line, wouldn't `focusScrollSpec` already just be "true" or "false" or whatever?  So splitting it on ":" and taking the first result seems unnecessary?  I may be reading the code wrong though...